metadata-ingestion/transformers.md

```diff
@@ -59,6 +66,32 @@ transformers:
       get_owners_to_add: "<your_module>.<your_function>"
 ```
 
+### Mark dataset status
+
+If you would like to stop a dataset from appearing in the UI then you need to mark the status of the dataset
+```yaml
+transformers:
+  - type: "mark_dataset_status"
```

Comment:
It seems a bit extreme to mark all datasets that are ingested by this source as removed. 
What is the specific use-case that we are trying to address here? 

The new `rollback` functionality of `datahub ingest` command might be a better way to clear out a bad ingestion?

Reply:
The specific use-case is when I am trying to remove all datasets. The transformer does not mean we mark all datasets as removed. The schema and table filters are present on all datasources. They can be used to filter out what we have to mark as removed. e.g. If I have a specific list of tables that  I want to remove then I can filter for them in a receipe and can mark them as removed. I'll add this in docs to explain when this can be removed.

Reply:
There have been some instances of people asking "How can we remove a dataset" on the slack channel. So those people can use this transformer after filtering for the specific datasets in the source and mark them as removed.

Reply:
There is also this now: https://datahubproject.io/docs/how/delete-metadata/#delete-by-urn